### PR TITLE
Version select

### PR DESCRIPTION
**Pull request by skliarzfort**
## PR Description
https://trello.com/c/zMyzBPlO/58-mobile-and-tablet-version-select
https://trello.com/c/eZxqGkFQ/59-tablet-content
https://trello.com/c/1j7etDne/60-mobile-content-without-tables
https://trello.com/c/nqFjN2hd/74-change-accordeons-design
https://trello.com/c/XHIUP24D/61-mobile-tables
https://trello.com/c/BlMv8KKE/68-adjust-table-width
https://trello.com/c/MbQx9xiX/77-add-class-wrapper-for-tables-and-images

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: css/components/_alt-nav.scss [hunk 1]
```diff
@@ -8,13 +8,49 @@
...
-        .alt-nav__drop {
-            display: block;
+        @include hover {
+            .alt-nav__drop {
+                display: block;
+            }
+        }
+    }
+
+    &.show {
```
Comment (FRITS-Kh) [inline css/components/_alt-nav.scss:18]: Please move this section below after `&.dropdown` section.

File: css/components/_alt-nav.scss [hunk 2]
```diff
@@ -81,5 +143,39 @@
...
+        width: 100%;
+        height: 100%;
+        background: rgba($black, .2);
+        z-index: 101;
+        display: none;
+        pointer-events: none;
+
+        @include media(">=desktop") {
+            display: none !important;
+        }
```
Comment (FRITS-Kh) [inline css/components/_alt-nav.scss:173]: Why do we need `important` here?
Also, it duplicates `display: none;` above.

File: css/components/_form-collapse.scss [hunk 3]
```diff
@@ -1,5 +1,9 @@
 .form-collapse {
     margin: 0 0 30px;
+
+    @include media("<tablet") {
+        margin: 0 0 20px;
```
Comment (FRITS-Kh) [inline css/components/_form-collapse.scss:5]:
```suggestion
        margin-bottom: 20px;
```

File: css/pages/_content.scss [hunk 4]
```diff
@@ -46,6 +67,59 @@
...
+            td,
+            th {
+                word-break: break-word;
+            }
+        }
+
+        &[width-100],
+        &.width-100 {
+            max-width: 100%;
+            width: 100%;
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:96]: Why do we duplicate widths if we have them by default?

File: css/pages/_content.scss [hunk 5]
```diff
@@ -46,6 +67,59 @@
...
+    tr {
+        @include media("<tablet") {
+            display: block;
+        }
+    }
+
+    thead {
+        tr {
+            @include media("<tablet") {
+                display: none !important;
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:120]: Also, why do we need `!important` here?

File: css/pages/_content.scss [hunk 6]
```diff
@@ -54,6 +128,47 @@
...
+                border-bottom-left-radius: 4px;
+                border-bottom-right-radius: 4px;
+                border-bottom: 0;
+            }
+
+            td:last-child {
+                @include media("<tablet") {
+                    border-bottom-left-radius: 4px;
+                    border-bottom-right-radius: 4px;
+                    border: 0;
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:166]:
```suggestion
                border-bottom: none;
            }

            td:last-child {
                @include media("<tablet") {
                    border-bottom-left-radius: 4px;
                    border-bottom-right-radius: 4px;
                    border: none;
```

File: css/pages/_content.scss [hunk 7]
```diff
@@ -83,12 +203,33 @@
...
         border-bottom: 2px solid $grey-3;
+
+        @include media("<tablet") {
+            padding-left: 40%;
+            border-bottom: 1px solid $grey-3;
+            overflow: hidden;
+            min-height: 45px;
+        }
+
+        &:before {
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:214]:
```suggestion
        &::before {
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: css/components/_alt-nav.scss (hunk 2 above)
Comment (skliarzfort) [reply css/components/_alt-nav.scss:173]: js will add inline display block on some resolution

File: css/pages/_content.scss (hunk 4 above)
Comment (skliarzfort) [reply css/pages/_content.scss:96]: it's specific classes for content manager.
table can have another default width due to js

File: css/pages/_content.scss [hunk 8]
```diff
@@ -54,6 +135,47 @@
...
+                background: $grey-lightest;
+            }
+        }
+
+        &:first-child {
+            @include media("<tablet") {
+                border-radius: 4px 4px 0 0;
+            }
+
+            td:first-child {
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:155]:
```suggestion
            &,
            td:first-child {
```

File: css/pages/_content.scss [hunk 9]
```diff
@@ -54,6 +135,47 @@
...
+        }
+
+        &:last-child {
+            @include media("<tablet") {
+                border-bottom-left-radius: 4px;
+                border-bottom-right-radius: 4px;
+                border-bottom: none;
+            }
+
+            td:last-child {
```
Comment (FRITS-Kh) [inline css/pages/_content.scss:169]:
```suggestion
            &,
            td:last-child {
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — APPROVED** [review]

**Pull request merged**